qlib/contrib/data/highfreq_handler.py

```diff
@@ -163,6 +249,45 @@ def get_feature_config(self):
         return fields, names
 
 
+class HighFreqGeneralBacktestHandler(HighFreqBacktestHandler):
```

Comment:
Should this class inherit from DataHandlerLP directly?

Both class HighFreqHandler and class HighFreqGeneralHandler are special cases of the high frequent handlers.
We don't have a base class for high frequent handlers. So it should inherit from DataHandlerLP directly